controllers/v1beta1/task_controller.go

```diff
@@ -592,43 +612,6 @@ func filterDynamicSecrets(secrets map[string]corev1.Secret) map[string]corev1.Se
 	return filteredSecrets
 }
 
-// check for the API/SSH settings in the task spec first, if nothing there use the one from our settings.
-func (r *LagoonTaskReconciler) getTaskValue(lagoonTask *lagoonv1beta1.LagoonTask, value string) string {
-	switch value {
-	case "TASK_API_HOST":
-		if lagoonTask.Spec.Task.APIHost == "" {
-			return r.LagoonAPIConfiguration.APIHost
-		}
-		return lagoonTask.Spec.Task.APIHost
-	case "TASK_SSH_HOST":
-		if lagoonTask.Spec.Task.SSHHost == "" {
-			return r.LagoonAPIConfiguration.SSHHost
-		}
-		return lagoonTask.Spec.Task.SSHHost
-	case "TASK_SSH_PORT":
-		if lagoonTask.Spec.Task.SSHPort == "" {
-			return r.LagoonAPIConfiguration.SSHPort
-		}
-		return lagoonTask.Spec.Task.SSHPort
-	case "LAGOON_CONFIG_API_HOST":
-		if lagoonTask.Spec.Task.APIHost == "" {
-			return r.LagoonAPIConfiguration.APIHost
-		}
-		return lagoonTask.Spec.Task.APIHost
-	case "LAGOON_CONFIG_SSH_HOST":
-		if lagoonTask.Spec.Task.SSHHost == "" {
-			return r.LagoonAPIConfiguration.SSHHost
-		}
-		return lagoonTask.Spec.Task.SSHHost
-	case "LAGOON_CONFIG_SSH_PORT":
-		if lagoonTask.Spec.Task.SSHPort == "" {
-			return r.LagoonAPIConfiguration.SSHPort
-		}
-		return lagoonTask.Spec.Task.SSHPort
-	}
-	return ""
-}
-
```

Comment:
I removed the fallback for extracting the `SSHPort` and `SSHHost` from the task spec because the lagoon API doesn't send these values in the task payload

Reply:
We can probably deprecate this from the lagoon api spec